webapp/locales/de.json

```diff
@@ -316,66 +316,67 @@
         "reports": {
             "empty": "Glückwunsch, es gibt nichts zu moderieren.",
             "name": "Meldungen",
-            "status": "Aktueller Status",
-            "content": "Inhalt",
-            "author": "Autor",
-            "decision": "Entscheidung",
-            "enabled": "Entsperrt",
-            "disabled": "Gesperrt",
-            "decided": "Entschieden",
-            "noDecision": "Keine Entscheidung!",
-            "decideButton": "Bestätige",
-            "DecisionSuccess": "Erfolgreich entschieden!",
-            "enabledBy": "Entsperrt von",
-            "disabledBy": "Gesperrt von",
-            "previousDecision": "Vorherige Entscheidung:",
-            "enabledAt": "Entsperrt am",
-            "disabledAt": "Gesperrt am",
+            "reporter": "gemeldet von",
+            "submitter": "gemeldet von",
+            "disabledBy": "deaktiviert von",
             "reasonCategory": "Kategorie",
             "reasonDescription": "Beschreibung",
-            "submitter": "Gemeldet von",
-            "numberOfUsers": "{count} Nutzern",
+            "status": "",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-31T14:56:52Z" title="Tuesday, December 31st 2019, 3:56:52 pm +01:00">Dec 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -316,66 +316,67 @@
         "reports": {
             "empty": "Glückwunsch, es gibt nichts zu moderieren.",
             "name": "Meldungen",
-            "status": "Aktueller Status",
-            "content": "Inhalt",
-            "author": "Autor",
-            "decision": "Entscheidung",
-            "enabled": "Entsperrt",
-            "disabled": "Gesperrt",
-            "decided": "Entschieden",
-            "noDecision": "Keine Entscheidung!",
-            "decideButton": "Bestätige",
-            "DecisionSuccess": "Erfolgreich entschieden!",
-            "enabledBy": "Entsperrt von",
-            "disabledBy": "Gesperrt von",
-            "previousDecision": "Vorherige Entscheidung:",
-            "enabledAt": "Entsperrt am",
-            "disabledAt": "Gesperrt am",
+            "reporter": "gemeldet von",
+            "submitter": "gemeldet von",
+            "disabledBy": "deaktiviert von",
             "reasonCategory": "Kategorie",
             "reasonDescription": "Beschreibung",
-            "submitter": "Gemeldet von",
-            "numberOfUsers": "{count} Nutzern",
+            "status": "",
```


I see a lot of code reverted in this PR. Have the translations not been updated on Lokalise?

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-01-17T12:23:14Z" title="Friday, January 17th 2020, 1:23:14 pm +01:00">Jan 17, 2020</time>_

---


We have to import both, English and German in Lokalise to avoid this. Thank you for being aware